### ISSUE TITLE
Article: What is required for WCAG 2.1

### ISSUE THREAD

**Issue by anevins12** [a11yproject/a11yproject.com#913]
## Description
With the era of automated testing there's a greater need to decipher testing results into genuine issues from false positives. Automated testing tools often report issues that are not necessarily failures of WCAG 2.1. Without being an accessibility expert, it seems there is no clear way to filter genuine conformance problems.

The main idea of the article is teaching everyone how they can check whether something is "normative"/ required in WCAG 2.1.

## Would you like to write the article or request that it be written?
I'll write it.

**Comment by anevins12** [comment]
Background:

* Pursuing long term accessibility investment can be an exhausting effort. While effort is inevidetable, prioritising our efforts can make the most of our resources and prevent us from becoming burnt-out.
* A starting point to accessibility can be checking whether a website or app meets the WCAG 2.1 level AA criteria. At this stage there are many automated tools available to you. While automated audit tools have their advantages, they can also direct our efforts in the wrong places. Automated audit tools can raise issues that are not required for WCAG conformance and sometimes issues that are not real. As a side note, automated tools only cover 20% to 50% of WCAG criteria and a human being is needed for the rest.
* Considering the goal of meeting WCAG conformance, issues that fall outside of that goal can be depriortised and tackled at a later stage.

Problem:
*

**Comment by anevins12** [comment]
@ericwbailey may I ask if you're okay with this article?

**Comment by ericwbailey** [comment]
Sounds good at a high level. Question back to you: do you think it could be done as shorter articles about common WCAG interpretation pitfalls?

**Comment by anevins12** [comment]
Absolutely @ericwbailey thank you for the direction!

**Comment by ericwbailey** [comment]
Closed for inactivity.
